README.md

```diff
@@ -179,6 +179,8 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/SSEK_PIUS)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/pius-ssekweyama-23665794)
 
+👤 **Original Design 1**
+- Original design idea by [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on Behance.
 
```

Comment:
- [ ] Please add these credits to the `Acknowledgments` section, not the `Authors` section as you did.